src/i18n/es.js

```diff
@@ -488,16 +488,25 @@ eliminación, seguridad, transferencias transfronterizas y otros temas.
     },
   },
   subscriber_gdpr: {
-    description: 'Aquí encontrarás todos los consentimientos dados por tu Suscriptor.',
+    description: 'Aquí encontrarás el historial de todos los consentimientos dados por tu Suscriptor.',
     empty_data: 'Este Suscriptor no ha aceptado ni rechazado ningun permiso.',
     empty_html_text: 'Sin texto legal definido',
     permission_description: 'Texto personalizado',
     permission_name: 'Nombre del campo',
     permission_value: 'Valor',
-    title: 'Estado RGPD del Suscriptor',
+    title: 'Historial de permisos RGPD del suscriptor',
```

Comment:
```suggestion
    title: 'Historial de permisos RGPD del Suscriptor',
```

Reply:
We use all Doppler words like Subscribers, Lists, Campaigns with the first capital letter.

Reply:
Got it